chainntnfs/txconfnotifier.go

```diff
@@ -470,7 +506,8 @@ func (tcn *TxConfNotifier) DisconnectTip(blockHeight uint32) error {
 	// clients is always non-blocking.
 	for initialHeight, txHashes := range tcn.txsByInitialHeight {
 		for txHash := range txHashes {
-			for _, ntfn := range tcn.confNotifications[txHash] {
+			confSet := tcn.confNotifications[txHash]
```

Comment:
any reason `!ok` check is skipped here?

Reply:
the assumption is that the keys in `txsByInitialHeight` are a subset of `confNotifications`, though it doesn't hurt to add the check